README.md

```diff
@@ -1,141 +1,133 @@
-    - [ ] Plugin creates an index with this name
-    - [ ] Plugin adds all items from the collection to the index
+**Supported Strapi versions**:
 
--> This create the interraction between strapi and MeiliSearch
+- Strapi v3.4.x
 
-### C
+(This plugin may work with the older Strapi versions, but these are not tested nor officially supported at this time.)
 
-- [ ] Possibility to add more than one collection (hear multiple indexes)
-- [ ] Add a listener on changes on all collections (ex if a restaurant is deleted it should be deleted from meilisearch as well)
+**Node / NPM versions**:
 
--> This create all basic needs of a user with MeiliSearch
+- NodeJS >= 12.10 <= 14
+- NPM >= 6.x
 
-### E (BONUS or open for debate)
+**We recommend always using the latest version of Strapi to start your new projects**.
 
-- [ ] Add customizable collections Relations.
-    Exemple: Two collections: Restaurants and Category. Category has a many to many relation with Restaurant. Automatic array creation should be created when indexing both. Example a Restaurant with its relation to Category
-```js
-{
-    id: '1'
-    name: 'La belle frite',
-    Category: ['Gras', 'Belge', 'Gastronomique']
-}
+## 🌎 Community support
 
-```
-And without
-```js
-{
-    id: '1'
-    name: 'La belle frite'
-}
+- For general help using Strapi, please refer to [the official Strapi documentation](https://strapi.io/documentation/).
+- Strapi Slack [channel](https://slack.strapi.io/)
 
-```
-## Consideration
+## Plugin's goal
 
-- [ ] adding a preview searchbar in the plugin page
+We want to create a Plugin that provides automatic actions and an interface on the strapi dashboard that helps the user start with MeiliSearch. The User should be able to create or remove new collections from MeiliSearch.
 
-## TODO
-- [ ] Steps on issues and notions
-- [ ] How do they handle testing ? 
+In the background, the plugin should re-index the documents that have been changed in the collections.<br>
+For example, If I change the name of the restaurant `Tonio` with `Tony` in a Strapi collection, an update should automatically be done to update it as well on MeiliSearch.
 
+No additional routes should be created on strapi for the front end users as they will be using the MeiliSearch API to search and not Strapi (unless we want to make this possible?).
 
-### Not in MVP
+## MVP
 
-- [ ] Settings on each index in a JSON string
-- [ ] Warning on re-indexation when changing Settings ? (find a way)
-- [ ] more friendly settings UI (still ugly but not a json string) attached to each index. 
+- Downloadable plugin
+- Communication with a running MeiliSearch
+- Interface with the following:
+    - Add host and keys
+    - Ability to chose which collections should be indexed in MeiliSearch
+    - Every time a collection is updated, the updated element should be updated in MeiliSearch as well (listener to add, edit and delete)
+    - Possibility to remove a collection from MeiliSearch
```

Comment:
I selected all the MVP part, not only the bonus one. I meant removing all the list and add a link to your issues that are certainly up to date :)

Reply:
Oh my bad